Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -330,7 +333,7 @@ WI.FontStyles = class FontStyles
         let inlineCSSStyleDeclaration = this._nodeStyles.inlineStyle;
         let properties = inlineCSSStyleDeclaration.visibleProperties;
 
-        cssProperty = properties.find(property => property.name === name);
+        cssProperty = properties.findLast(property => property.name === name && property.enabled);
```

Comment:
`WI.CSSProperty.enabled` is not what you're looking for:

https://github.com/WebKit/WebKit/blob/54140a7b23efcb75955d08e7e7882371d1af3d99/Source/WebInspectorUI/UserInterface/Models/CSSProperty.js#L411-L412

It's likely `WI.CSSProperty.overriden`.

Reply:
I tried overridden, but only enabled works.

Reply:
&& property.enabled has been working, not && property.overridden because it switches between the two font-weights and will go to the overridden one which is not what we want